src/Library/demos/Action Bar/code.rs

```diff
@@ -0,0 +1,21 @@
+use crate::workbench;
+use gtk::prelude::*;
+
+pub fn main() {
+    let action_bar: gtk::ActionBar = workbench::builder().object("action_bar").unwrap();
+    let button: gtk::ToggleButton = workbench::builder().object("button").unwrap();
+    let start_widget: gtk::Button = workbench::builder().object("start_widget").unwrap();
+    let end_widget: gtk::Button = workbench::builder().object("end_widget").unwrap();
+
+    button.connect_clicked(move |button| {
```

Comment:
You should use this function https://gtk-rs.org/gtk-rs-core/stable/0.18/docs/glib/object/trait.ObjectExt.html#tymethod.connect_notify_local with parameter "active" instead

Reply:
> You should use this function https://gtk-rs.org/gtk-rs-core/stable/0.18/docs/glib/object/trait.ObjectExt.html#tymethod.connect_notify_local with parameter "active" instead

Made the changes :)